lupin/fields/polymorphic_list.py

```diff
@@ -16,15 +16,18 @@ class PolymorphicList(Field):
                             "painting": painting_schema
                         })
     """
-    def __init__(self, on, schemas, **kwargs):
+    def __init__(self, on, schemas, default_on=None, **kwargs):
```

Comment:
```suggestion
    def __init__(self, on, schemas, default_type=None, **kwargs):
```

Reply:
well, in general we use "type" as value of th "on" parameter : 
f.PolymorphicObject(on="type",...)
I think you have this usage in mind when you propose "default_type" 
but if use on other field, 'default_type' is less logical
f.PolymorphicObject(on="kind", default_type="jewel", )

Reply:
For me the `on` parameter define the name of the field that contains the type to instantiate.
So having `on` defining the name of the field and `default_on` containing the default value of that field is kind of misleading.